src/utils.js

```diff
@@ -4,15 +4,19 @@ import styled from 'styled-components';
 export const themeGet = (lookup, fallback) => ({ theme } = {}) => get(theme, lookup, fallback);
 
 export const getComponentVariant = (theme, componentName, variant) => {
-  const config = themeGet(`components.${componentName}.variants.${variant}`, theme.variants[componentName][variant])({
+  const config = themeGet(`variants.${componentName}.${variant}`, theme.variants[componentName][variant])({
```

Comment:
We are now using the same thing for the lookup and the fallback?

Reply:
@cehsu I bumped the major version